README.md

```diff
@@ -167,23 +169,48 @@ Use [`func (tf *TF) Cmd(name string, args ...string) *exec.Cmd`](https://pkg.go.
 
 ### Verbose mode
 
-Enable verbose output using the `-v` CLI flag. It works similar to `go test -v`. When enabled, the whole output will be streamed. If disabled, only logs from failed task are send to the output.
+In verbose mode, the whole output will be streamed. If disabled, only logs from failed task are send to the output.
 
-Use [`func (*TF) Verbose`](https://pkg.go.dev/github.com/pellared/taskflow#TF.Verbose) to check if verbose mode was set within the task's command.
+By default, verbose mode is turned on.
+It can be controlled by registering a boolean parameter and assigning it to the field `Taskflow.Verbose`.
+
+This is also provided via the convenience function `taskflow.VerboseParam(*taskflow.Taskflow)`, which needs to be explicitly called.
+The convenience function registers a boolean parameter `verbose`, short `v`, and assigns it to the `Taskflow` field.
+This parameter defaults to `false` by default, so as soon as this parameter is registered, verbose mode has to be explicitly enabled via the CLI.
+This then works similar to `go test -v`. 
+
+You can then also re-use the registered parameter to check if verbose mode was set within the task's command.
 
 ### Default task
 
 Default task can be assigned via `DefaultTask` field in [`type Taskflow`](https://pkg.go.dev/github.com/pellared/taskflow#Taskflow).
 
-When default task is set, then it is run if no task is provied via CLI.
+When default task is set, then it is run if no task is provided via CLI.
 
 ### Parameters
 
-The parameters can be set via CLI using the `key=val` syntax after CLI flags. For example, `go run ./build -v ci=true all` would run the `all` task with `ci` parameter set to `"true"` in verbose mode.
+The parameters can be set via CLI using flag syntax. Flags must be registered in "long form", and have an optional short form using only one letter.
```

Comment:
It might be good to describe what happens if there is no description for a flag. TBH I have even not checked how it works. Do you think it would make sense to implement it as a "hidden" parameter? This is how a task without description works currently.

Reply:
I understand that some tasks will be hidden to allow reusable procedures that shouldn't be explicitly called by the user on the CLI.
However, having a hidden parameter that can only be set through CLI is a weird form of hiding. After all, this is a tool for (build) automation where you have direct access to the source, I'd like to see what a parameter does if needed.